### DISCUSSION TITLE
where oobabooga_chat history is  stored?

### DISCUSSION THREAD

**Discussion by panmaster** [oobabooga/text-generation-webui#2542] [General]
The issue is  that  even if  i clear  the  history , at  next launch of oobabooga  I see old text  history,lookos like clearing history does not work

**Comment by ArachneRomBor** [comment]
I think clearing chat history works.
In any case, the chat history is stored in the `logs` directory.

**Comment by panmaster** [comment]
thank you

**Reply by snfwr7142** [reply]
حته اني مالكيت سجل الدردشه

**Comment by ArachneRomBor** [comment]
A month ago, I claimed to the questioner here that the chat history is saved in the `logs` directory. However, this is clearly no longer true. The chat history probably stopped being written to the `logs` directory after one of the previous updates. The question is why?

**Reply by c-hoffmann** [reply]
...and also, where is it stored now?

**Comment by Ascentions** [comment]
text-generation-webui\user_data\logs\chat---text-generation-webui\user_data\logs\instruct---text-generation-webui\user_data\logs\notebook
